pkg/globalmanager/lifelonglearningjob.go

```diff
@@ -675,6 +675,7 @@ func (jc *LifelongLearningJobController) createInferPod(job *sednav1.LifelongLea
 			URL: &MountURL{
 				URL:    inferModelURL,
 				Secret: jobSecret,
+				Mode:   workerMountWriteOnly,
```

Comment:
Why does inferModel need to be writable?

Reply:
define new mode: workerMountDownloadOnly